learn/wavemaker-release-notes/v10-7-2.md

```diff
@@ -0,0 +1,85 @@
+---
+title: "WaveMaker 10.7.2 - Release date: 21 June 2021"
+id: ""
+sidebar_label: "Release v10.7.2"
+---
+*WaveMaker releases an update with bug fixes.*
+
+## Improvements
+
+---
+
+- Improved the format of the **Currency** and **Number** widgets to support natural and financial modes by setting the **Input Mode** property.
+- For the **Upload widget**, added the ability to aptly highlight the dropzone when using the drag and drop approach.
+
+## Bug Fixes
+
+---
+
+- Fixed an issue related to the `mvn clean install` build failing for the projects stored in D or E drives.
```

Comment:
Fixed an issue related to project build failing in windows machine when the project is located in drives other than the C drive.